CHANGELOG.md

```diff
@@ -14,7 +14,10 @@ _None_
 
 ### Bug Fixes
 
-_None_
+* Fix issue with Storyboard templates when compiled with Swift 4.2 on macOS.  
```

Comment:
I try to, when applicable, prefix my changelog entries with the command, so in this case `IB: `. The issue also only applies to `segues-swift4`, so we may want to note that.